### DISCUSSION TITLE
Webhook  相关，POST方式 有机会添加一个输入框 输入json格式的内容 自定义RequestBody吗

### DISCUSSION THREAD

**Discussion by MidnightShake** [jxxghp/MoviePilot#789] [问题咨询]
插件 Webhook  相关，POST方式 有机会添加一个输入框 输入json格式的内容 自定义RequestBody吗，比如基础提供一个 标题变量 和 信息内容变量 作为调用，添加一个测试按钮，兼容更多其他第三方Webhook

**Comment by serenity666** [comment]
附议，这样就可以支持企业微信群机器人了
